### ISSUE TITLE
Accessibility support in txt via txt.Accessibility

### ISSUE THREAD

**Issue by diverted247** [diverted247/txtjs#10]
Add class to manage fallback DOMText elements within canvas element.

txt.Accessibility.addText( text:string ):number;
// Allows HTML to be passed direct

txt.Accessibility.addText( text:string , priority:number ):number; 
//Priority wraps text and re-orders elements H1 > H2 > H3 > P

txt.Accessibility.removeText( id:number );

txt.Accessibility.removeAll();

//Constants for priority. Will result in outer tag wrapping contents
txt.Accessibility.H1 = 0;
txt.Accessibility.H2 = 1;
txt.Accessibility.H3 = 2;
txt.Accessibility.P = 3;

Priority allows many different text elements to denote DOM element hierarchy based on size. When addText or removeText is called, all values within the canvas fallback text will update.

Accessibility support will lean on support for fallback DOM Text within a canvas element. If higher level Accessibility APIS exist this class will remap into them as needed.

The txt.Text, txt.CharacterText, txt.PathText will push text values into the txt.Accessibility API as they are added to canvas or updated. Depending on the text.size, they will map into h1,h2,p elements to represent the designed hierarchy of the canvas view. This will be on by default.

The API can also be used directly by an app developer to enhance what is presented to the screenreader.

**Comment by diverted247** [comment]
Here is a preview of txt.Accessibility class.

http://code.tedpatrick.com/txtjs_accessibility/

By default, it will place the .text value in a 'p' element. If .accessibilityText is set, that will be used rather than the .text value. Additionally .accessibilityPriority allows a developer to control the order of the DOM elements within the canvas element. Writing DOM contents is deferred as typically there are several written at once and we only want 1 DOM change per set of text value changes. Elements removed from the EaselJS displaylist, will be removed from canvas DOM elements in parallel. The goal is to make sure the Canvas element always reflects what is on canvas automatically or with added developer context.

Feedback welcome.

**Comment by stevefaulkner** [comment]
thanks!

--

Regards

SteveF
HTML 5.1 <http://www.w3.org/html/wg/drafts/html/master/>
